runtime/src/iree/hal/local/plugins/registration/init.c

```diff
@@ -16,6 +17,13 @@
 #include "iree/hal/local/plugins/embedded_elf_plugin.h"
 #endif  // IREE_HAVE_HAL_EXECUTABLE_EMBEDDED_ELF_PLUGIN
 
+// TODO: figure out a better place for this
```

Comment:
I agree. If the plugin can accept an option that would be nice. If we don't want to touch the iree runtime, we may want to use an environmental variable. Looks good for now.